examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -107,32 +129,65 @@
-    cut_coords=cut_coords, threshold=threshold, title=title)
-plotting.show()
 
-###########################################################################
+###############################################################################
 # Now, we compute the (corrected) p-values with a permutation test.
+#
+# .. important::
+#   In this example, ``threshold`` is set to 0.001, which enables
+#   cluster-level inference.
+#   Performing cluster-level inference will increase the computation time of
+#   the permutation procedure.
+#   Increasing the number of parallel jobs (``n_jobs``) can reduce the time
+#   cost.
 from nilearn.glm.second_level import non_parametric_inference
-neg_log_pvals_permuted_ols_unmasked = \
-    non_parametric_inference(second_level_input,
-                             design_matrix=design_matrix,
-                             model_intercept=True, n_perm=1000,
-                             two_sided_test=False,
-                             smoothing_fwhm=8.0, n_jobs=1)
-
-###########################################################################
+
+out_dict = non_parametric_inference(
+    second_level_input,
+    design_matrix=design_matrix,
+    model_intercept=True,
+    n_perm=500,  # 500 for the sake of time. Ideally, this should be 10,000.
+    two_sided_test=False,
+    smoothing_fwhm=8.0,
+    n_jobs=1,
+    threshold=0.001,
+)
+
+###############################################################################
 # Let us plot the (corrected) negative log p-values for the nonparametric test.
-title = ('Group left-right button press: \n'
-         'permutation test (FWER < 10%)')
-display = plotting.plot_glass_brain(
-    neg_log_pvals_permuted_ols_unmasked, colorbar=True, vmax=3,
-    display_mode='z', plot_abs=False, cut_coords=cut_coords,
-    threshold=threshold, title=title)
-plotting.show()
+IMAGES = [
+    neg_log_pval,
+    out_dict['logp_max_t'],
+    out_dict['logp_max_size'],
+    out_dict['logp_max_mass'],
+]
+TITLES = [
+    "Parametric Test",
+    "Permutation Test\n(Voxel-Level Error Control)",
+    "Permutation Test\n(Cluster-Size Error Control)",
+    "Permutation Test\n(Cluster-Mass Error Control)",
+]
+
+fig, axes = plt.subplots(figsize=(8, 8), nrows=2, ncols=2)
+img_counter = 0
+for i_row in range(2):
+    for j_col in range(2):
+        ax = axes[i_row, j_col]
+        plotting.plot_glass_brain(
+            IMAGES[img_counter],
+            colorbar=True,
+            vmax=3,
+            display_mode='z',
+            plot_abs=False,
+            cut_coords=cut_coords,
+            threshold=threshold,
+            figure=fig,
+            axes=ax,
+        )
+        ax.set_title(TITLES[img_counter])
+        img_counter += 1
+
+fig.suptitle("Group left-right button press\n(negative log10 p-values)")
 
 # The neg-log p-values obtained with nonparametric testing are capped at 3
 # since the number of permutations is 1e3.
```

Comment:
I've added the plt.show(). I also reorganize the comments a little bit, since they no longer matched up well with the new plot. The new vmax is 2.7 to match the new number of permutations.